src/customDaoInfo.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { ReactElement } from "react";
+
+interface IDaoCustomInfo {
+  [network: string]: { [address: string]: ReactElement | string };
+}
+
+const data: IDaoCustomInfo = {
+  ganache: {
+  },
+  rinkeby: {
+    "0x9003196e314e03dae65f0bf26788befc714d68a1": (
+      <>
+        <div>Welcome to The Venus Project DAO, a decentralized organization built on DAOstack.</div>
```

Comment:
does the description need to be with these tags ?
can it be just text?

Reply:
Just text would be extremely limited.

Reply:
If you are want to eliminate react.js components like `<Link`, then you lose important features like being able to cleanly navigate to the Plugin Proposals page.

Reply:
What is the issue?

Reply:
BTW, if a DAO wants just text, then they can.  Tags are not required.

Reply:
I think that having a simple description text is cleaner and might be used as dao description in other pages , with different formats.. and even not in alchemy. 
@roienatan @dkent600 ?

Reply:
Depends how much the link for example is important to us. It's also possible to use plain text and add a "GO TO" button or a link near it which directs us to the url.

Reply:
The link might be important , though the description can be formatted as dao.json :
{
  "description": "the mother of all daos",
  "url": "www.blala.org"
}
Giving it another thought , we might consider to index that at subgraph level.,as dao metadata,. and dapps(alchemy) could fetch that from there.
This will be easy , and relative fast to implement (in relation to the dao header page feature). 
We will take this approach in alchemy 2.0.

Reply:
I would suggest:

That the needs for enforcing a structured landing page, where DAOstack defines the structure, should be enumerated before implementing.  For example, it might facilitate the user in entering very simple custom data, and might facilitate searching for specific data, such as a website URL, using metadata to refine the search.

And that those needs should be weighed against what believe would be the futility of predicting a structure acceptable to all DAOs.

I think some combination of meta-structure and absolute flexibility would be ideal.